docs/v3/automate/add-schedules.mdx

```diff
@@ -65,7 +65,26 @@ If using work pool-based deployments, the `deploy` method has the same schedule-
 
 When `my_flow` is served with this interval schedule, it will run every 10 minutes beginning at midnight on January, 1, 2026 in the `America/Chicago` timezone:
 
-```python
+<CodeGroup>
+```python prefect >= 3.1.16
+from datetime import timedelta, datetime
+from prefect.schedules import Interval
+
+from myproject.flows import my_flow
+
+my_flow.serve(
+  name="flowing",
+  schedules=[
```

Comment:
I think for an example with a single schedule you should use the `schedule=` (singular) kwarg instead